### PR TITLE
Minor changes in combinatorics

### PR DESCRIPTION
**Pull request by vishalg2235**
Minor changes (typos) in combinatorics.

### CODE REVIEW

File: sympy/combinatorics/perm_groups.py [hunk 1]
```diff
@@ -1895,7 +1895,7 @@ def is_primitive(self, randomized=True):
     def minimal_blocks(self, randomized=True):
         '''
         For a transitive group, return the list of all minimal
-        block systems. If a group is intrasitive, return `False`.
+        block systems. If a group is intransitive, return `False`.
```
Comment (gxyd) [inline sympy/combinatorics/perm_groups.py]: I'm not sure if the term 'intransitive' is used or 'non transitive' is used.

Comment (jksuom) [reply sympy/combinatorics/perm_groups.py]: 'intransitive' seems to be used in some books on group theory.

Comment (gxyd) [reply sympy/combinatorics/perm_groups.py]: Agreed. Its even used in the Handbook that way.

Comment (vishalg2235) [reply sympy/combinatorics/perm_groups.py]: I think 'intransitive' is right.

**Pull request merged**